docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,123 @@
+rules from the Customer Networks that need to be defined to allow customer applications to
+connect to Aiven services. This depends on the cloud provider being used, and the
+customer’s network architecture.
+
+| Source                   | Destination     | Description                                                                                                                             |
+| ------------------------ | ----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet           | Workload subnet | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet           | Workload subnet | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Your remote applications | Workload subnet | Varies depending on services being used.                                                                                                |
+
+### Traffic from the workload
+
+The workload subnet nodes communicate with each other. The nature of these communications
+varies depending on the services, features, and plugins being used.
+
+| Source          | Destination              | Description                                                                                                                |
+| --------------- | -------------------------| -------------------------------------------------------------------------------------------------------------------------- |
+| Workload subnet | Aiven Management         | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes |
+| Workload subnet | Aiven Management         | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                    |
+| Workload subnet | AWS CloudFront           | Communication channel for downloading RPM packages for bastion and workload nodes setup                                    |
+| Workload subnet | Workload subnet          | Intra-subnet communication between Aiven nodes. Ports are dynamic and dependent on services, features, and plugins used.   |
+| Workload subnet | Your remote applications | Varies depending on services being used.                                                                                   |
+| Workload subnet | DNS & NTP                | Destination dependant on cloud provider                                                                                    |
+
+:::note
+Aiven Management IPs are considered relatively stable but do occasionally change and are
+considered dynamic for the purposes of firewall policies. To that end, an egress of
+`0.0.0.0/0` is generally required to accommodate traffic to Aiven Management.
```

Comment:
FWIW this equally applies to AWS Cloudfront IPs, we could also want to mention that as well here.